x-pack/plugins/security_solution/public/management/hooks/endpoint/use_get_endpoints_list.test.ts

```diff
@@ -107,4 +109,141 @@ describe('useGetEndpointsList hook', () => {
       })
     );
   });
+
+  it('should also list inactive agents', async () => {
+    const getApiResponse = apiMocks.responseProvider.metadataList.getMockImplementation();
+
+    // set a few of the agents as inactive/unenrolled
+    apiMocks.responseProvider.metadataList.mockImplementation(() => {
+      if (getApiResponse) {
+        return {
+          ...getApiResponse(),
+          data: getApiResponse().data.map((item, i) => {
+            const isInactiveIndex = [0, 1, 3].includes(i);
+            return {
+              ...item,
+              host_status: isInactiveIndex ? HostStatus.INACTIVE : item.host_status,
+              metadata: {
+                ...item.metadata,
+                host: {
+                  ...item.metadata.host,
+                  hostname: isInactiveIndex
+                    ? `${item.metadata.host.hostname}-inactive`
+                    : item.metadata.host.hostname,
+                },
+                Endpoint: {
+                  ...item.metadata.Endpoint,
+                  status: isInactiveIndex
+                    ? EndpointStatus.unenrolled
+                    : item.metadata.Endpoint.status,
+                },
+              },
+            };
+          }),
+        };
+      }
+      throw new Error('some error');
+    });
+
+    // verify metadata list does indeed have inactive agents
+    expect(
```

Comment:
is this needed here? It seems to be testing the api mock and not the actual UI list.

Reply:
Yeah quite right! It's not really needed here. I just added these just to be sure while testing locally. Will remove.

Reply:
done c3995250eb233b9b0ff3bcf948e1311d1950a237